lib/qiita/markdown/filters/sanitize.rb

```diff
@@ -47,6 +47,7 @@ def node
           attributes: {
             "a" => [
               "href",
+              "rel",
```

Comment:
`ExternalLink` filter adds the rel=nofollow attribute. But, after executing the `Sanitize` filter, the `Sanitize` filter removes all rel attributes. Therefore, added the `rel` to the rule to keep `rel` attributes.